addons/web/static/src/core/utils/autoresize.js

```diff
@@ -99,9 +99,7 @@ export function resizeTextArea(textarea, options = {}) {
         borderTopWidth: 0,
         borderBottomWidth: 0,
         paddingTop: 0,
-        paddingRight: style.paddingRight,
         paddingBottom: 0,
-        paddingLeft: style.paddingLeft,
```

Comment:
@juliusc2066 :wave: Do you know why these were added? Is this safe?

Otherwise I can always force the style via xpath but it's a bit :/